parser/tests/unit/test_parser_unit.py

```diff
@@ -1,30 +1,47 @@
+import os
 import pathlib
 
+from neo4j import GraphDatabase
+
 from openfoodfacts_taxonomy_parser import normalizer, parser
 
 # taxonomy in text format : test.txt
 TEST_TAXONOMY_TXT = str(pathlib.Path(__file__).parent.parent / "data" / "test.txt")
 
 
 def test_normalized_filename():
-    x = parser.Parser()
+
+    # Initialize neo4j
+    uri = os.environ.get("NEO4J_URI", "bolt://localhost:7687")
+    driver = GraphDatabase.driver(uri)
+    session = driver.session()
```

Comment:
in case you care, it could have been written as a [pytest fixture](https://docs.pytest.org/en/latest/how-to/fixtures.html) :-)

In `conftest.py`:

```python
import pytest

@pytest.fixture
def graph_session():
    uri = os.environ.get("NEO4J_URI", "bolt://localhost:7687")
    driver = GraphDatabase.driver(uri)
    session = driver.session()
    try:
        yield session
    finally:
        session.close()
```

And then in each test, just get a graph_session parameter, that will give you the session:

```python
def test_normalized_filename(graph_session):
    x = parser.Parser(graph_session)
    ...
    # no need to close
```

Reply:
That's cool, I'll refactor it accordingly